--- buildroot/support/scripts/mkusers	2017-08-09 13:58:16.284577209 +0200
+++ custom/support/scripts/mkusers	2017-08-08 15:47:08.594460033 +0200
@@ -302,7 +302,9 @@
     fi
 
     # Remove any previous instance of this user
-    for _f in "${PASSWD}" "${SHADOW}"; do
+    # recalbox doesn't use the /etc/shadow
+    # for _f in "${PASSWD}" "${SHADOW}"; do
+    for _f in "${PASSWD}"; do
         sed -r -i -e '/^'"${username}"':.*/d;' "${_f}"
     done
 
@@ -336,9 +338,10 @@
            "${username}" "${uid}" "${_gid}"     \
            "${comment}" "${_home}" "${_shell}"  \
            >>"${PASSWD}"
-    printf "%s:%s:::::::\n"      \
-           "${username}" "${_passwd}"   \
-           >>"${SHADOW}"
+    # recalbox doesn't use the /etc/shadow
+    #printf "%s:%s:::::::\n"      \
+    #       "${username}" "${_passwd}"   \
+    #       >>"${SHADOW}"
 
     # Add the user to its additional groups
     if [ "${groups}" != "-" ]; then
