--- a/Makefile	2020-01-19 19:24:32.261293488 +0100
+++ b/Makefile2	2020-01-19 19:24:47.641415492 +0100
@@ -69,6 +69,7 @@
 
 TARGET_NAME := mupen64plus_next
 CC_AS ?= $(CC)
+NASM ?= nasm
 
 GIT_VERSION ?= " $(shell git rev-parse --short HEAD || echo unknown)"
 ifneq ($(GIT_VERSION)," unknown")
@@ -507,7 +508,7 @@
 	$(STRINGS) "$<" | $(TR) -d '\r' | $(AWK) -v dest_dir="$(AWK_DEST_DIR)" -f $(CORE_DIR)/tools/gen_asm_defines.awk
 
 %.o: %.asm $(AWK_DEST_DIR)/asm_defines_gas.h
-	nasm -i$(AWK_DEST_DIR)/ $(ASFLAGS) $< -o $@
+	$(NASM) -i$(AWK_DEST_DIR)/ $(ASFLAGS) $< -o $@
 
 %.o: %.S $(AWK_DEST_DIR)/asm_defines_gas.h
 	$(CC_AS) $(CFLAGS) -c $< -o $@
