--- a/Config/Supermodel.ini	2018-01-25 22:03:43.000000000 +0100
+++ b/Config/Supermodel.ini	2020-01-10 10:54:33.040479830 +0100
@@ -41,14 +41,14 @@
 addr_out = "127.0.0.1"
 
 ; Common 
-InputStart1 = "KEY_1,JOY1_BUTTON9"
-InputStart2 = "KEY_2,JOY2_BUTTON9"
-InputCoin1 = "KEY_3,JOY1_BUTTON10"
-InputCoin2 = "KEY_4,JOY2_BUTTON10"
-InputServiceA = "KEY_5"
-InputServiceB = "KEY_7"
-InputTestA = "KEY_6"
-InputTestB = "KEY_8"
+InputStart1 = "KEY_1,JOY1_BUTTON10"
+InputStart2 = "KEY_2,JOY2_BUTTON10"
+InputCoin1 = "KEY_3,JOY1_BUTTON9"
+InputCoin2 = "KEY_4,JOY2_BUTTON9"
+InputServiceA = "KEY_5,JOY1_BUTTON12"
+InputServiceB = "KEY_7,JOY2_BUTTON12"
+InputTestA = "KEY_6,JOY1_BUTTON13"
+InputTestB = "KEY_8,JOY2_BUTTON13"
 
 ; 4-way digital joysticks
 InputJoyUp = "KEY_UP,JOY1_UP"
@@ -90,8 +90,8 @@
 InputSteering = "JOY1_XAXIS"        ; analog, full steering range
 
 ; Pedals
-InputAccelerator = "KEY_UP,JOY1_UP"
-InputBrake = "KEY_DOWN,JOY1_DOWN"
+InputAccelerator = "KEY_UP,JOY1_BUTTON8"
+InputBrake = "KEY_DOWN,JOY1_BUTTON7"
 
 ; Up/down shifter manual transmission (all racers)
 InputGearShiftUp = "KEY_Y"           ; sequential shift up
