--- buildroot/package/odroid-scripts/odroid-scripts.hash	2017-08-09 13:58:16.284577209 +0200
+++ custom/package/odroid-scripts/odroid-scripts.hash	2017-08-08 15:47:08.594460033 +0200
@@ -1,2 +1,2 @@
 #  Locally computed hash
-sha256 779e8beccf8c66d51e495a4cc0e6eccf72c295499a1ecdc0740aa4459de786a0 odroid-scripts-a252de04562dcf8d8a4918a544b45a9e3d46d2fb.tar.gz
+sha256 9d61472ee6aed2d1a7b5afdcb91b823003c8524a242572f24f89d4e57323daf3  odroid-scripts-cfd289b1d69136b8bfd725d90cd648da4654f2bd.tar.gz
