--- buildroot/package/kodi-visualisation-shadertoy/Config.in	2017-08-09 13:58:16.284577209 +0200
+++ custom/package/kodi-visualisation-shadertoy/Config.in	2017-08-08 15:47:08.594460033 +0200
@@ -1,5 +1,6 @@
 config BR2_PACKAGE_KODI_VISUALISATION_SHADERTOY
 	bool "kodi-visualisation-shadertoy"
+    depends on BR2_PACKAGE_KODI
 	select BR2_PACKAGE_LIBGLEW if BR2_PACKAGE_HAS_LIBGL
 	select BR2_PACKAGE_LIBPLATFORM
 	help
